corehq/apps/reports/templates/reports/form/partials/single_form.html

```diff
@@ -174,7 +191,7 @@
             {% endif %}
             {% if user.can_edit_data %}
             <p>
-                {% if not is_archived %}
+                {% if not is_archived and not edit_info.was_edited %}
```

Comment:
an edited form already doesn't contribute to the case or show up in exports so there shouldn't need to be a reason to archive it.

also, if you unarchive an archived form it was going to be a bit more annoying to check whether it was an edit or not and then properly set the `doc_type` - so it was a bit of me being lazy